pkg/cmd/server/kubernetes/master.go

```diff
@@ -76,10 +76,12 @@ func (c *MasterConfig) RunPersistentVolumeClaimBinder() {
 }
 
 func (c *MasterConfig) RunPersistentVolumeClaimRecycler(recyclerImageName string, client *client.Client) {
+	privilged := true
```

Comment:
Was it determined that this pod actually needs privileged to work correctly and not just run as UID 0?  If so, do we still need the host mount only SCC?  If we're keeping the host mount only SCC and not using it for the recycler we need to update the description annotation.